docs/swarm/index.md

```diff
@@ -10,14 +10,14 @@ parent="engine_swarm"
 weight="1"
 +++
 <![end-metadata]-->
-# Docker Swarm overview
+# Docker Swarm mode overview
```

Comment:
For some reason `Swarm Mode` looks more correct to me than `Swarm mode`, but I can't really articulate why. I guess since we're treating `Swarm` as a proper noun, `Swarm Mode` feels like it should also be one.

Feel free to disregard if you think the current capitalization is correct.